components/raftstore/src/store/simple_write.rs

```diff
@@ -231,7 +231,7 @@ pub struct SimpleWriteReqDecoder<'a> {
 impl<'a> SimpleWriteReqDecoder<'a> {
     pub fn new(
         fallback: impl FnOnce(&'a [u8], u64, u64) -> RaftCmdRequest,
-        logger: &Logger,
+        logger: Option<&Logger>,
```

Comment:
Seems logger is only used to provide some context value of target region, so what about replace it with something like `region_id` or `peer_id`?

Reply:
Neither `region_id` nor `peer_id` are available in the callers.